mysite/settings.py

```diff
@@ -78,20 +78,28 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.\
-            UserAttributeSimilarityValidator',
+        'NAME': (
```

Comment:
Looks like you forgot to rebase on top of `origin/master` (did you update your local repo from github remote with `git fetch --all` or `git pull --rebase`?).